pkg/apiserver/archive/archive.go

```diff
@@ -249,3 +250,38 @@ func (s *Service) report(c *gin.Context) {
 
 	c.JSON(http.StatusOK, report)
 }
+
+// @Summary Delete the specified archived experiment.
+// @Description Delete the specified archived experiment.
+// @Tags archives
+// @Produce json
+// @Param uid path string true "uid"
+// @Success 200 "delete ok"
+// @Failure 500 {object} utils.APIError
+// @Router /experiments/{uid} [delete]
+func (s *Service) delete(c *gin.Context) {
+	var (
+		err error
+		exp *core.Experiment
+	)
+
+	uid := c.Param("uid")
+
+	if exp, err = s.archive.FindByUID(context.Background(), uid); err != nil {
+		if !gorm.IsRecordNotFoundError(err) {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
+		} else {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInvalidRequest.New("the archived experiment is not found"))
+		}
```

Comment:
```suggestion
		if gorm.IsRecordNotFoundError(err) {
			c.Status(http.StatusInternalServerError)
			_ = c.Error(utils.ErrInvalidRequest.New("the archived experiment is not found"))
		} else {
			c.Status(http.StatusInternalServerError)
			_ = c.Error(utils.ErrInternalServer.WrapWithNoMessage(err))
		}
```

Reply:
how about updating to this? 😃

Reply:
OK, because it is written in other places as well. I will revise these places.